Assets/Scripts/Cards/DisplayCard.cs

```diff
@@ -2,30 +2,33 @@
 using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.UI;
+using TMPro;
 
 public class DisplayCard : MonoBehaviour {
 
 	public Card card;
 
-	public Text name;
-	public Text description;
+	public TextMeshProUGUI cardName;
+	public TextMeshProUGUI type;
+	public TextMeshProUGUI abilities;
 
-	public Text mana;
-	public Text attack;
-	public Text health;
+	public TextMeshProUGUI mana;
+	public TextMeshProUGUI attack;
+	public TextMeshProUGUI health;
 
 	public Image cardArt;
 
 	// Use this for initialization
 	void Start () {
-		name.text = card.name;
-		description.text = card.description;
+		cardName.text  = card.cardName;
```

Comment:
These tabs are really starting to get to me, we should have a conversation in Discord about getting everyone on tabs or spaces and not both 😄

Reply:
Hah that drives me crazy too (I use 2 char tabs locally). Is that something we can settle with a .gitattributes file?